### ISSUE TITLE
import cve's automatically with a queue where CVe's can be reviewed

### ISSUE THREAD

**Issue by jelly** [archlinux/arch-security-tracker#141]
It would be nice to import CVE's automatically using an API maybe such as https://cve.circl.lu/api/ and add a review / queue to the security tracker to review these issues.

**Comment by jelly** [comment]
@ysf can you take a look at this?

**Comment by ysf** [comment]
It would be great to have a sanitized dump to get a feel of how the data has to look later on and generally work with the tracker.

**Comment by shibumi** [comment]
Hi @ysf, sad that we didn't met on 36C3 :)
You can find dumps of our data via putting `.json` in your URL bar. 
For example for a CVE:
`https://security.archlinux.org/CVE-2019-6474.json`

For an AVG:
`https://security.archlinux.org/AVG-1032.json`

and so on. Hope this helps.

**Comment by ysf** [comment]
Yeah, but I don't want to scrape those and add them under an anonymous user into my local database if those could be easily sanitized to get a realistic working environment. Actually it's a time issue, if it's quicker to scrape and import than to sanitize and share I'll gladly do it.

**Comment by Foxboron** [comment]
I think there should be two views on the tracker for this data.

* One view that parses the recent/modified list and contains the following data
   * CVE
   * pkgname
   * version
   * severity

Obviously the entries needs to be as few false positives as possible. We should also enable easy creation of an AVG group and do some basic matching on `pkgname` and `version` to find existing AVGs.

* One view to search CVEs which are reserved, and not public, so we can have a todolist when details are filled from the recently modified list
